dispatcher/DAQController.py

```diff
@@ -89,161 +87,90 @@ def SolveProblem(self, latest_status, goal_state):
                 self.one_detector_arming = True
```

Comment:
Does this mean that if we are in this state, we cannot do the can force stop? I.e. should we add:
```python
self.can_force_stop[det] = False
````

Reply:
`can_force_stop` and `one_detector_arming` are independent control checks